packages/web/modals/validator-next-step.tsx

```diff
@@ -60,6 +63,10 @@ export const ValidatorNextStepModal: FunctionComponent<
               className="w-full"
               mode="primary-bullish"
               onClick={() => {
```

Comment:
Maybe we should extract this to a these onClick functions to a handleClick using useCallback. It's better perf since it won't trigger as many child rerenders

Reply:
Similar to what you do in validator-squad below